build.gradle

```
@@ -55,8 +55,10 @@ dependencies {
     annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
 
     // Lombok
-    compileOnly 'org.projectlombok:lombok'
+    implementation 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
+    testCompileOnly 'org.projectlombok:lombok'
+    testAnnotationProcessor 'org.projectlombok:lombok'
```

이거 test 라고 따로 안붙여도 알아서 추가 되지 않나요?
```suggestion
    implementation 'org.projectlombok:lombok'
    annotationProcessor 'org.projectlombok:lombok'
```